terraform/environment/region/ecs_cluster.tf

```diff
@@ -120,6 +120,10 @@ locals {
       name  = "LAY_REPORT_CSV_FILENAME",
       value = local.lay_report_csv_file
     },
+    {
+      name  = "RUN_ONE_OFF_MIGRATIONS",
+      value = "true"
```

Comment:
So there's a bit more to do here so we can make it environment specific so we can release to pre and test then update the var for prod.

What you need to do is add the variable run_one_off_migrations into the environment folders terraform.tfvars.json for every environment block in there (follow example of the other vars in there). Next add it to the mapping which is in variables.tf.

Finally update the value on line 125 to be var.account.run_one_off_migrations.

This way we can stagger out the changes and test it in pre without having to intervent with manual terraform changes etc..